### PR TITLE
Add DerpDEX to dex.trades for zkSync

### PR DESCRIPTION
**Pull request by lgingerich**

### CODE REVIEW

**Comment by lgingerich** [comment]
@jeff-dude I'm not sure what this error means. I've rechecked all my work but can't see anything wrong

```
Compilation Error
  [WARNING]: Did not find matching node for patch with name 'derpdex_v1_zksync_base_trades' in the 'models' section of file 'models/_sector/dex/trades/zksync/_schema.yml'
```

File: models/_sector/dex/trades/zksync/platforms/derpdex_zksync_base_trades.sql
Comment (Hosuke) [inline models/_sector/dex/trades/zksync/platforms/derpdex_zksync_base_trades.sql:1]: The filename need to be matched to `derpdex_v1_zksync_base_trades` in the schema file.

**Comment by Hosuke** [comment]
Hi @lgingerich,
please resolve the conflicts with the main branch in this PR as well.
Thank you.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @lgingerich 

[Sample derpdex_v1_zksync_base_trades]()

**Review by jeff-dude — APPROVED** [review]

**Pull request merged**